integrate_test.sh

```diff
@@ -33,34 +33,12 @@ echo "travis pull request repo slug -> ${GITHUB_REPOSITORY}"
 echo "travis pull request actor -> ${GITHUB_ACTOR}"
 echo "travis pull request repo param -> $1"
 
+# clone dubbo-go-samples
+git clone -b 3.0 https://github.com/apache/dubbo-go-samples.git samples
```

Comment:
i think it should not change the branch.

Reply:
thanks for your review, now different branches of dubbo-go will use the dubbo-go-samples of the corresponding branch to test, and the default test scheme has been added.